roles/srv-announcer/defaults/main.yml

```diff
@@ -0,0 +1,51 @@
+# NOTE: variables set here can - and potentially should - be overridden when invoking the role
+
+
+# INFO: The following variables are MANDATORY or will be tested for existence
+
+# There are three different ways to define which version is going to be installed.
+# All of them are mutual exclusive.
+# 1.
+# announce_artifact_file_path: local path in filesystem
+#
+# 2.
+# announcer_version: string of a specific release version
+#
+# 3.
+# announce_artifact_file_url: URL
+# announce_artifact_checksum: value
+
+# DOCS: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html
+# announcer_aws_key_id: String
+# announcer_aws_access_key: String
+# announcer_aws_region: String
+
+# announcer_zone_domain: String, e.g. domain.tld
+
+# NOTE: The srv-announcer can only manage one entry in the SRV record list, but the role allows
+#       to define multiple entries. For each entry a new systemd unit is created. To also
+#       facilitate the usecase of removing a certain unit file, each block allows to define the
+#       attribute `deprecated` in order to indicate a unit removal. After running a play with
+#       that attribute, the whole block can be safely removed
+# EXAMPLE:
+# ```yaml
+#  mx:
+#    name: _mx._tcp.sub.
+#    target: mx1.sub.domain.tld
+#  metrics:
+#    name: _metrics._tcp.sub.
+#    target: monitoring.sub.domain.tld
+#    deprecated: true
+# ```
+announcer_srv_records: {}
+
+
+# INFO: The following variables are NOT mandatory and won't be tested for existence
+
+# NOTE: in seconds
+announcer_record_ttl: 60
+# NOTE: relative values
+announcer_record_priority: 10
```

Comment:
Perhaps we can keep priority at 0 for consistency? Or is there a motivation for this value to be different to how we use priority in other places?

Reply:
I did't look into other places. I just wanted to reflect the [defaults](https://github.com/wireapp/srv-announcer/blob/master/main.go#L30-L107).